harness/determined/tensorboard/base.py

```diff
@@ -61,3 +63,18 @@ def delete(self) -> None:
         Delete all objects from the backing persistent storage.
         """
         pass
+
+
+def get_metric_writer() -> tensorboard.BatchMetricWriter:
```

Comment:
Is it possible to just return an `EventFileWriter` or `SummaryWriter`? The current implementation looks not very clean.